CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-12-3)
+
+- Added `replaceDefaultTitle` and `replaceDefaultDescription` options to the SkyErrorTitleComponent and SkyErrorDescriptionComponent. [#7](https://github.com/blackbaud/skyux-errors/pull/7)
```

Comment:
What do you think of de-emphasizing the names of the properties and components and focusing on what the new functionality enables? I'm thinking something along the lines of: 
"Added options to overwrite the default title and default description on error components to enable error messages to display standard images with custom text."

Reply:
sounds good to me